test/e2e/ratelimit/ratelimit_universal.go

```diff
@@ -136,9 +136,9 @@ conf:
 		Expect(err).ToNot(HaveOccurred())
 
 		// demo-client specific RateLimit works
-		Eventually(verifyRateLimit("demo-client", 5), "60s", "1s").Should(Equal(4))
+		Eventually(verifyRateLimit("demo-client", 5), "60s", "10s").Should(Equal(4))
 		// verify determinism by running it once again with shorter timeout
-		Eventually(verifyRateLimit("demo-client", 5), "30s", "1s").Should(Equal(4))
+		Eventually(verifyRateLimit("demo-client", 5), "30s", "10s").Should(Equal(4))
```

Comment:
The usage of Eventually here seems unnecessary no? Is it to deal with stuff in between bucket window?

Reply:
May be, yeah. I assumed it was to deal with the indeterminism of when the bucket starts filling/how long sending 5 requests takes?